test/Renci.SshNet.IntegrationTests/SftpClientTests.cs

```diff
@@ -83,8 +83,8 @@ public async Task Create_directory_with_contents_and_list_it_async()
                 actualFiles.Add((file.FullName, file.IsRegularFile, file.IsDirectory));
             }
 
-            _sftpClient.DeleteFile(testFilePath);
-            _sftpClient.DeleteDirectory(testDirectory);
+            await _sftpClient.DeleteFileAsync(testFilePath, CancellationToken.None);
+            await _sftpClient.DeleteDirectoryAsync(testDirectory, CancellationToken.None);
```

Comment:
I know you are mirroring the sync version, but I don't think it's really testing deletion. If not adding a separate test, perhaps at least another call to ListDirectory/Exists here? e.g.

 ```suggestion
            await _sftpClient.DeleteFileAsync(testFilePath, CancellationToken.None);
            
            // check the file no longer exists
            // ...
            
            await _sftpClient.DeleteDirectoryAsync(testDirectory, CancellationToken.None);
            
            // check the directory no longer exists
            // ...
```

Reply:
Yes agreed, this was a bit lazy on my part. I will work on a new test to verify async delete functionality.